src/styles/blocks/container.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+  margin: 0 auto;
+  max-width: 1020px;
+  padding: 0 25px;
+
+  @include onTablet {
+    padding: 0 50px;
+  }
+
+  @include onDesktop {
+    padding: 0;
+  }
```

Comment:
its bad practice to have padding 0 for container, looks like content sticks to borders 
![image](https://user-images.githubusercontent.com/108153052/209403355-c18b778d-1068-4021-afac-bd273bde8021.png)